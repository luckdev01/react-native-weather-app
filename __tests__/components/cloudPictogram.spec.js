import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';
import CloudPictogram from '../../src/components/cloudPictogram';
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

describe('<CloudPictogram />', () => {
    it('Input and Button component renders correctly', () => {
      const tree = renderer.create(
        <CloudPictogram weatherIcon={2} />
      );
      const json = tree.toJSON();
      expect(json).toMatchSnapshot();
    });
    const wrapper = shallow(<CloudPictogram weatherIcon={2} />);

    it('Should exists', () => {
        expect(wrapper.length).toEqual(1);
    });
});
