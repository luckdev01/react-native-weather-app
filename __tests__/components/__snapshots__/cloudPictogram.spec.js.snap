exports[`<CloudPictogram /> Input and Button component renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "marginTop": 150,
    }
  } />
`;
